promql/parser/generated_parser.y

```diff
@@ -641,18 +666,42 @@ series_item     : BLANK
                 | series_value TIMES uint
                         {
                         $$ = []SequenceValue{}
+                        // add an additional value for time 0, which we ignore in tests
                         for i:=uint64(0); i <= $3; i++{
                                 $$ = append($$, SequenceValue{Value: $1})
                         }
                         }
                 | series_value signed_number TIMES uint
                         {
                         $$ = []SequenceValue{}
+                        // add an additional value for time 0, which we ignore in tests
                         for i:=uint64(0); i <= $4; i++{
                                 $$ = append($$, SequenceValue{Value: $1})
                                 $1 += $2
                         }
                         }
+                // Histogram descriptions (part of unit testing)
+                | histogram_desc_set
```

Comment:
I'm wondering if the naming should be more consistent with `series_value` above. Something like `histogram_series_value`